package.json

```diff
@@ -106,6 +111,8 @@
     "start-deved": "npm run webpack && web-ext run -s build --firefox=firefoxdeveloperedition & npm run webpack:watch",
     "test": "npm-run-all test:*",
     "test:karma": "webpack --config webpack.test-unit.js && NODE_ENV=test karma start",
+    "test:ui": "test/integration/setup-webext.sh && npm run ui-tests-build && mocha ./test/dist/integration --retries 1 --no-deprecation --reporter=list --compilers js:babel-core/register --require babel-polyfill",
```

Comment:
@jrbenny35 I get the following 

```
 Error: The geckodriver executable could not be found on the current PATH. Please download the latest version from https://github.com/mozilla/geckodriver/releases/ and ensure it can be found on your PATH.
```

Could we get something like https://github.com/vladikoff/node-geckodriver into the process so it "just works" when I run `test:ui` ?

Reply:
That is assuming the user is running on Nightly as it seems that plugin keeps up with the most recent releases without a way to install an older version. I'd rather point the user to the geckodriver install and they can install whichever version they need according to their Firefox version.

We could do a docker image, or something like that maybe.

Reply:
Hm , I think `geckodriver 0.21.0` is backwards compatible, our other Selenium infrastructure didn't have issues running in Stable or Nightly. Especially last couple of versions, don't think they have too many changes

Reply:
@jrbenny35 any ideas about:

```
FIREFOX_BINARY=/Applications/FirefoxNightly.app/Contents/MacOS/firefox npm run test:ui

> notes@4.3.0-dev test:ui /Users/vladikoff/mozilla/notes
> test/integration/setup-webext.sh && npm run ui-tests-build && mocha ./test/dist/integration --retries 1 --no-deprecation --reporter=list --compilers js:babel-core/register --require babel-polyfill

Webextension built and moved to root dir.

> notes@4.3.0-dev ui-tests-build /Users/vladikoff/mozilla/notes
> babel ./test/integration --quiet --no-babelrc --presets=env -d ./test/dist/integration


  1) The Firefox Notes web extension "before each" hook for "should have a default note named correctly"

  0 passing (3s)
  1 failing

  1) The Firefox Notes web extension "before each" hook for "should have a default note named correctly":
     WebDriverError: Could not install add-on: /var/folders/cv/w3cp7lkj6bq70z90s51jllsc0000gn/T/addon-ee6daf6a-c8ae-4451-9bdc-2893432b63f5.xpi: ERROR_SIGNEDSTATE_REQUIRED: The addon must be signed and isn't.
      at Object.throwDecodedError (node_modules/selenium-webdriver/lib/error.js:550:15)
      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:542:13)
      at Executor.execute (node_modules/selenium-webdriver/lib/http.js:468:26)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)


```

Reply:
It's not starting Nightly for me locally

Reply:
I installed the stuff package.json. i.e. "selenium-webdriver": "^4.0.0-alpha.1",

do we need some kind of a selenium launcher or do i need to manually run it?

Reply:
That's interesting as I have the preference set to ignore signatures.

Reply:
This doesn't look right though: ```WebDriverError: Could not install add-on: /var/folders/cv/w3cp7lkj6bq70z90s51jllsc0000gn/T/addon-ee6daf6a-c8ae-4451-9bdc-2893432b63f5.xpi```, the xpi should be named ```firefox_notes.xpi```.